gen/drive/DifferentialDrive.yml

```diff
@@ -57,7 +57,7 @@ classes:
                 self.drive = wpilib.drive.DifferentialDrive(self.left, self.right)
 
       Six motor drivetrain::
-        import wpilib
+
```

Comment:
The example needs `import wpilib` to work?

Reply:
It's already imported on the line below.

Reply:
... heh, since when does that work?

Well, put it back anyways. If you don't import wpilib separately, autocomplete breaks.

Reply:
> autocomplete breaks

It works fine with Jedi for me. What IDE/configuration are you using?

Reply:
vscode

Reply:
fine I don't care